<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExcelExport</class>
 <widget class="QMainWindow" name="ExcelExport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ExcelExport</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="OutputView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="OpenButton">
    <property name="geometry">
     <rect>
      <x>402</x>
      <y>411</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GenerationButton">
    <property name="geometry">
     <rect>
      <x>486</x>
      <y>411</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>生成</string>
    </property>
   </widget>
   <widget class="QDockWidget" name="ListView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>211</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(191, 191, 191);</string>
    </property>
    <widget class="QWidget" name="ListViewContents"/>
   </widget>
   <widget class="QDockWidget" name="DetailView">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>0</y>
      <width>421</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(191, 191, 191);</string>
    </property>
    <widget class="QWidget" name="ssss_2">
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>421</width>
        <height>361</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>419</width>
         <height>359</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="checkButton">
    <property name="geometry">
     <rect>
      <x>571</x>
      <y>411</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>排错</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>658</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionGenerateDir"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="actionField"/>
    <addaction name="actionBatchGenerator"/>
    <addaction name="actionFilterSetting"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSetting"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionField">
   <property name="text">
    <string>字段设置</string>
   </property>
  </action>
  <action name="actionBatchGenerator">
   <property name="text">
    <string>批量生成</string>
   </property>
  </action>
  <action name="actionGenerateDir">
   <property name="text">
    <string>设置生成路径</string>
   </property>
  </action>
  <action name="actionFilterSetting">
   <property name="text">
    <string>过滤器设置</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
